luigi/server.py

```diff
@@ -214,7 +214,11 @@ def get(self, path):
 class RootPathHandler(tornado.web.RequestHandler):
 
     def get(self):
-        self.redirect("/static/visualiser/index.html")
+        visualization_graph = configuration.get_config().get('core', 'visualization_graph', '')
```

Comment:
Ah, nah. I just meant that `configuration.get_config().get('core', 'visualization_graph', '')` can be replaced with `core().visualization_graph`. But that's probably a bad idea as then the server needs to import the client code. Which doesn't make sense. :)

Ignore me :)